helm/alfresco-content-services/Chart.yaml

```diff
@@ -93,7 +93,7 @@ dependencies:
   - name: kibana
     alias: kibana-audit
     repository: https://helm.elastic.co
-    version: 7.17.3
+    version: 8.5.1
```

Comment:
Not sure about bumping this chart. Maybe we should stay on the same version as elasticsearch but considering it doesnt break anything in workflow it should be fine. Im more worried about pipeline envs